exercises/I-array-properties/exercise.js

```diff
@@ -6,8 +6,12 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
-}
+   if (arr.length === 0) {
```

Comment:
what does ``arr.length === 0`` return? It is either true or false. So you can simplify this code by returning the condition straight away.